guides/source/upgrading_ruby_on_rails.md

```diff
@@ -237,6 +237,10 @@ The `protected_attributes` gem is no longer supported in Rails 5.
 
 The `activerecord-deprecated_finders` gem is no longer supported in Rails 5.
 
+### Removed support for `rails-deprecated_sanitizer` gem
+
+The `rails-deprecated_sanitizer` gem is no longer supported in Rails 5, the [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer) gem should be used instead.
```

Comment:
Thank you for the pull request but I'll support this gem since we are still using in Shopify.